src/data/faq.json

```
@@ -135,15 +135,15 @@
                                 "active": false,
                                 "textblock": [
                                     "Even after the end of the development work on the CWA, you can still access your already scanned digital certificates and you can use the contact diary. However, you will not be able to scan any more certificates or to renew existing certificates. You will also no longer receive notifications about the certificates. Additionally, you can still view statistics of the Pandemic Radar; you can find a corresponding link on the status screen of the app.",
-                                    "A reduction in functionality will be implemented in Release 3.2, but this will not be activated until May 1, 2023. Up to and including April 30, 2023, releases 3.1 and 3.2 of the app will therefore not differ in functionality."
+                                    "A reduction in functionality has been implemented in Release 3.2, but this will not be activated until May 1, 2023. Up to and including April 30, 2023, releases 3.1 and 3.2 of the app will therefore not differ in functionality."
                                 ]
                             },
                             {
                                 "title": "Can I still obtain test results/digital test certificates via the app?",
                                 "anchor": "ramp_down_tests",
                                 "active": false,
                                 "textblock": [
-                                    "No, this will also no longer be possible from April 20, 2023 onwards, since the connection of the test centers to the CWA infrastructure is no longer available. To retrieve test results and lab reports, please use the app of the respective provider or laboratory."
+                                    "No, this is no longer be possible from April 20, 2023 onwards, as the infrastructure has been shut down on this date. To retrieve test results and lab reports, please use the app of the respective provider or laboratory."

```

"No, this is no longer be possible from April 20, 2023 onwards, as the infrastructure has been shut down on this date."

That is not quite grammatically correct and it can be simplified anyway.

"No, this is not possible since April 20, 2023, as the infrastructure was shut down on this date."

---

Does this also sound good to you @MikeMcC399?

```suggestion
                                    "No, this is not possible anymore since April 20, 2023, as the infrastructure was shut down on this date. To retrieve test results and lab reports, please use the app of the respective provider or laboratory."
```

---

Your suggestion is correct and longer. It does not convey any more information than my suggestion. I would choose the shorter version, however this is a matter of taste.